e2e/test/scenarios/visualizations-charts/pie_chart.cy.spec.js

```diff
@@ -66,6 +66,30 @@ describe("scenarios > visualizations > pie chart", () => {
     ].map(args => checkLegendItemAriaCurrent(args[0], args[1]));
   });
 
+  it("should not truncate legend titles when enabling percentages (metabase#48207)", () => {
+    visitQuestionAdhoc({
+      dataset_query: testQuery,
+      display: "pie",
+      visualization_settings: {
+        "pie.percent_visibility": "off",
+      },
+    });
+
+    cy.findByTestId("viz-settings-button").click();
+
+    leftSidebar().within(() => {
+      cy.findByText("Display").click();
+      cy.findByText("In legend").click();
+    });
+
+    cy.findByTestId("chart-legend").within(() => {
+      cy.findByText("Widget").then(([element]) => {
+        // When text is truncated, offsetWidth will be less than scrollWidth
```

Comment:
From https://stackoverflow.com/questions/72113890/cypress-how-to-check-if-the-element-has-a-cropped-text-by-text-overflow

Unfortunately we can't just check if the ellipses character appears, Cypress won't know that since it's not in the dom